code/src/main/kotlin/com/expediagroup/sdk/v2/core/client/util/GsonFactoryUtil.kt

```diff
@@ -0,0 +1,18 @@
+package com.expediagroup.sdk.v2.core.client.util
+
+import com.google.api.client.json.gson.GsonFactory
+
+/**
+ * Creates a GsonFactory instance using the provided builder or returns the default instance if the builder is null.
+ *
+ * TODO: To be extended to hold Gson Factory configuration
+ *
+ * @param builder An optional builder for customizing the GsonFactory instance. If null, the default GsonFactory instance is returned.
+ * @return An instance of GsonFactory based on the builder configuration or the default configuration.
+ */
+internal fun createGsonFactory(builder: GsonFactory.Builder? = null) =
```

Comment:
Aren't we using Jackson?

Reply:
Yes, but we still need to utilize a `Gson` instance. Plus, I am keeping this in a separate file since I am sure we will add logic here as soon as we carry on out migration to `Gson`.